src/mmw/js/src/analyze/controllers.js

```diff
@@ -55,8 +57,11 @@ var AnalyzeController = {
             });
 
         App.state.set('current_page_title', 'Geospatial Analysis');
-
         App.rootView.footerRegion.show(analyzeWindow);
+
+        analyzeWindow.model.get('aoiCensus').done(function(aoiCensus) {
+            App.aoiCensus = aoiCensus;
```

Comment:
I think we should try to avoid attaching more things to `App` and try to attach to `App.project` or something similar if we need to persist this across pages.